src/waveform/waveformwidgetfactory.cpp

```diff
@@ -71,7 +71,8 @@ WaveformWidgetFactory::WaveformWidgetFactory() :
         m_vsyncThread(NULL),
         m_frameCnt(0),
         m_actualFrameRate(0),
-        m_vSyncType(0) {
+        m_vSyncType(0),
+        m_playMarkerPosition(0.5) {
```

Comment:
can we use here: WaveformWidgetRenderer::s_DefaultPlayMarkerPosition; 
The constant should be probably moved.